test/v2/pid.cpp

```diff
@@ -12,4 +12,8 @@ BOOST_AUTO_TEST_CASE(test_pid)
 {
     namespace bp2 = boost::process::v2;
     BOOST_CHECK_NE(bp2::current_pid(), static_cast<bp2::pid_type>(0));
+
+    auto all = bp2::all_pids();
+    auto itr = std::find(all.begin(), all.end(), bp2::current_pid());
```

Comment:
This should have two more tests: one for a child process and one for a grand child.

Reply:
I won't be able to do that until I add my parent and child from pid functions. This pr is meant just for getting the exe path from a pid. If you are ok with me conflating the two pull requests I can do that if you want.

Reply:
In the meantime, I'm going to create a test for getting exe from path out of all pids.